src/libcore/result.rs

```diff
@@ -685,7 +685,7 @@ impl<T, E: fmt::Debug> Result<T, E> {
         match self {
             Ok(t) => t,
             Err(e) =>
-                panic!("called `Result::unwrap()` on an `Err` value: {:?}", e)
+                unwrap_failed("called `Result::unwrap()` on an `Err` value: {:?}", e),
```

Comment:
Oh I think that the trailing `: {:?}` part of the string can be removed in both cases, that's covered by the format string at the very bottom